.github/workflows/canary-integration-test.yml

```diff
@@ -78,7 +78,12 @@ jobs:
     - name: wait for ceph to be ready
       run: |
         tests/scripts/validate_cluster.sh
-        kubectl -n rook-ceph get pods
```

Comment:
IIRC the output in these tests is super long. Will all that lengthy output be in the artifacts now, with only the test output directly in the console?

Reply:
Right, all that lengthy output be in the artifacts now, with only the test output directly in the console.

Done.

Reply:
Done.